src/vs/editor/browser/widget/multiDiffEditorWidget/colors.ts

```diff
@@ -8,13 +8,13 @@ import { registerColor } from 'vs/platform/theme/common/colorRegistry';
 
 export const multiDiffEditorHeaderBackground = registerColor(
 	'multiDiffEditor.headerBackground',
-	{ dark: '#808080', light: '#b4b4b4', hcDark: '#808080', hcLight: '#b4b4b4', },
+	{ dark: '#262626', light: 'tab.inactiveBackground', hcDark: 'tab.inactiveBackground', hcLight: 'tab.inactiveBackground', },
 	localize('multiDiffEditor.headerBackground', 'The background color of the diff editor\'s header')
 );
 
 export const multiDiffEditorBackground = registerColor(
 	'multiDiffEditor.background',
-	{ dark: '#000000', light: '#e5e5e5', hcDark: '#000000', hcLight: '#e5e5e5', },
+	{ dark: 'editorBackground', light: 'editorBackground', hcDark: 'editorBackground', hcLight: 'editorBackground', },
```

Comment:
I think this should fix https://github.com/microsoft/vscode/issues/203940 ? cc @hediet

Reply:
yes